src/code/fv-components/fv-change-sex-toggle.js

```diff
@@ -28,7 +28,7 @@ const FVChangeSexToggle = ({sex, style={}, onChange}) => {
 
   return (
     <div className='geniblocks change-sex-toggle-group'>
-      <div className={classNames('sex-label female-label', { selected: isFemaleSelected })}>
+      <div className={classNames('sex-label female-label', { selected: isFemaleSelected })} onClick={handleSexToggle}>
```

Comment:
As we discussed separately, I believe the behavior in GV1 was that clicking on the "male" label would set the sex to male or have no effect, i.e. it didn't simply toggle. It's worth double-checking what the desired behavior should be and whether consistency with GV1 is a priority.